README.md

```diff
@@ -3,40 +3,85 @@
 > Official GitHub Action for [OSSF scorecard](https://github.com/ossf/scorecard).
 
 ## Installation
-The simplest and quickest way to install Scorecards's GitHub action is from the [GitHub's marketplace](https://github.com/marketplace/actions/ossf-scorecard-action).
-**WARNING**: the action does *not* support private repositories.
+**NOTE**: The Scorecards GitHub Action does not support private repositories. Private repositories can run Scorecards manually by following the [standard installation instructions)[https://github.com/ossf/scorecard#installation].
 
-### Inputs
+To install the Scorecards GitHub Action, you need to:
 
-| Name | Required | Description |
-| ----- | -------- | ----------- |
-| `result_file` | yes | The file that contains the results. |
-| `result_format` | yes | The format in which to store the results [json \| sarif]. For GitHub's scanning dashboard, select `sarif`. |
-| `repo_token` | yes | PAT token with read-only access. Follow [these steps](#pat-token-creation) to create it. |
-| `publish_results` | recommended | This will allow you to display a badge on your repository to show off your hard work (release scheduled for Q2'22). See details [here](#publishing-results).|
+1) Create a Personal Access Token (PAT) for authentication and save the token value as a repository secret; 
+    
+    (Note: current Scorecards users can reuse their existing PAT for the repository secret.)
+    
+3) Set up the workflow via the GitHub UI
 
-### PAT token creation
-1. Create a PAT token [here](https://github.com/settings/tokens/new) with the following read permissions:
+### Authentication
+1. [Create a Personal Access Token](https://github.com/settings/tokens/new) with the following read permissions:
     - Note: `Read-only token for OSSF Scorecard Action`
```

Comment:
since it seems hard to re-use an existing token (UX does not allow reading its value), users may often create a new token. If all tokens have the same name it will be confusing. Shall we tweak the name to include the repo name, e.g., `Read-only token for OSSF Scorecard Action - myorg/myrepo` or `Read-only token for OSSF Scorecard Action - repo:myrepo` ?

Reply:
That makes sense. Do we expect people to want to install Scorecards on many repos, say dozens or hundreds? This step could get onerous if so, but I'm not sure whether that's a reasonable expected use case for the target user.

Reply:
I expect some users to use it on multiple repos, but there's no tool to scale it to hundreds of repos atm. There may be a way to automate it via a tool we write (create token, save as secrets and create pull requests), but it does not exist today